.ci/scripts/generate-debug-docs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+gitUrl=$(git remote get-url origin)
+commit=$(git rev-parse HEAD)
```

Comment:
Potentially to be changed for the already defined env variables which are provided by the MBP or the gitHubEnv step. :)